File: backend/GymTurf.Domain/Entities/Booking.cs
using System;
using GymTurf.Domain.Common;

namespace GymTurf.Domain.Entities;

public class Booking : BaseEntity
{
    public Guid FacilityId { get; set; }
    public Guid UserId { get; set; }
    public string MemberName { get; set; } = string.Empty;
    public DateTime BookingDate { get; set; }
    public TimeSpan StartTime { get; set; }
    public TimeSpan EndTime { get; set; }
    public string Status { get; set; } = "Confirmed"; // Confirmed, Cancelled
    public string Notes { get; set; } = string.Empty;
}

File: frontend/src/types/booking.ts
export interface Booking {
  id: string;
  facilityId: string;
  facilityName: string;
  facilityType: string;
  userId: string;
  memberName: string;
  bookingDate: string;
  startTime: string; // "HH:mm"
  endTime: string;   // "HH:mm"
  status: string;    // "Confirmed" | "Cancelled"
  notes?: string;
}
